models/Glossary.php

```diff
@@ -27,7 +27,7 @@ class Glossary extends AbstractModel
     /**
      * @internal
      *
-     * @var int|null
+     * @var int
      */
     protected $id;
```

Comment:
Can be null for new unsaved models. Also for the getter and other models.
```
$glossary = new Glossary();
var_dump($glossary->getId());
```

Reply:
yes, you are right. Reverted the changes for BC reasons. thanks!